BrowserKit/Sources/ComponentLibrary/BottomSheet/BottomSheetViewController.swift

```diff
@@ -60,18 +60,21 @@ public class BottomSheetViewController: UIViewController,
     private lazy var scrollContentView: UIView = .build { _ in }
     private var contentViewBottomConstraint: NSLayoutConstraint!
     private var viewTranslation = CGPoint(x: 0, y: 0)
+    private var windowUUID: WindowUUID
```

Comment:
I might be missing something but it looks like this is set during initialization and never changed, can we update this to `let`?

Reply:
+1